psydac/linalg/block.py

```diff
@@ -85,6 +86,44 @@ def zeros(self):
         """
         return BlockVector(self, [Vi.zeros() for Vi in self._spaces])
 
+    # ...
+    def inner(self, x, y):
+        """
+        Evaluate the inner vector product between two vectors of this space V.
```

Comment:
same question (which space V?)

Btw, this docstring seems to be a copy of the abstract `inner` method, seems redundant.

Reply:
Same answer.

It is indeed a copy of the docstring of the abstract class, which will not show up since we have overridden the method. In order to see any docstring when you write `W.inner?` in IPython, or `help(W.inner)` in Python, the docstring of `BlockVectorSpace.inner` must be given explicitly.

Reply:
Ok, that does not seem very practical, but if that's the usual way...

Reply:
Indeed this could be error prone, but I have no better solution right now...